File: exercise/java/week06/exercise2/src/main/java/report/ReportData.java
package report;

public class ReportData {
    private int id;
    private double value;
    private String description;

    public ReportData(int id, double value, String description) {
        this.id = id;
        this.value = value;
        this.description = description;
    }

    public int getId() {
        return id;
    }

    public double getValue() {
        return value;
    }

    public String getDescription() {
        return description;
    }
}

File: exercise/ts/week06-legacy-code/exercise2/src/reportData.ts
export class ReportData {
    private readonly id: number;
    private readonly value: number;
    private readonly description: string;

    constructor(id: number, value: number, description: string) {
        this.id = id;
        this.value = value;
        this.description = description;
    }

    getId(): number {
        return this.id;
    }

    getValue(): number {
        return this.value;
    }

    getDescription(): string {
        return this.description;
    }
}